modules/tinymce/src/core/main/ts/api/Mode.ts

```diff
@@ -12,7 +12,7 @@ import Editor from './Editor';
 
 export interface EditorMode {
   /**
-   * Checks if the editor is in a readonly state.
+   * Checks if the editor content is in a readonly state.
```

Comment:
```suggestion
   * Checks if the editor is in a readonly state.
```
Assuming we're keeping this the same